data/sql/updates/pending_db_world/rev_1610238998819516081.sql

```diff
@@ -0,0 +1,323 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1610238998819516081');
+
+UPDATE `creature` SET `MovementType`=2, `currentwaypoint`=1 WHERE `guid`=13990;
+UPDATE `creature` SET `MovementType`=2, `currentwaypoint`=1 WHERE `guid`=13991;
+UPDATE `creature` SET `MovementType`=2, `currentwaypoint`=1 WHERE `guid`=13992;
+
+DELETE FROM `creature_addon` WHERE (`guid` IN (13990, 13991, 13992));
+INSERT INTO `creature_addon` (`guid`, `path_id`, `mount`, `bytes1`, `bytes2`, `emote`, `isLarge`, `auras`) VALUES
+(13991, 139910, 0, 0, 4097, 0, 0, NULL),
+(13992, 139920, 0, 0, 4097, 0, 0, NULL),
+(13990, 139900, 0, 0, 4097, 0, 0, NULL);
+
+SET @diff_x1 = 1;
+SET @diff_x2 = 1.85;
+SET @diff_y1 = 1.73; -- -1.73
```

Comment:
what's. the meaning of this comment?

Reply:
the value should be negative, but you can't assign to a SQL variable a negative value so I will subtract it later below instead of "add" like the others value, just a reminder